core/utils.py

```diff
@@ -88,18 +88,31 @@ def read_manual(fua: int, proj_crs: str | int | pyproj.CRS) -> geopandas.GeoData
     )
 
 
+def read_osmnx(
+    fua: int | str, proj_crs: str | int | pyproj.CRS
+) -> geopandas.GeoDataFrame:
+    """Read OSM roads from parquet format; return bare columns."""
+    return geopandas.read_parquet(_fua_path(fua, "osmnx")).to_crs(proj_crs)
+
+
 def read_parenx(
-    fua: int, option: str, proj_crs: str | int | pyproj.CRS
+    fua: int, proj_crs: str | int | pyproj.CRS, option: str
 ) -> geopandas.GeoDataFrame:
     """Read in prepared parenx data."""
 
     return (
-        geopandas.read_parquet(_fua_path(fua, "parenx", option=option))
+        geopandas.read_parquet(_fua_path(fua, f"parenx-{option}"))
         .explode(ingore_index=True, index_parts=False)
         .to_crs(proj_crs)
     )
 
 
+def read_sgeop(fua: int, proj_crs: str | int | pyproj.CRS) -> geopandas.GeoDataFrame:
```

Comment:
Should go ahead and rename to `neatnet` here? (and in data directories?)

Reply:
updating this PR now,  & can take care of this renaming part

Reply:
@jGaboardi taking this back :D martin will take care of the renaming in both repos at one go (later on) so sticking with `sgeop` for now